stories/rank-step.stories.jsx

```diff
@@ -0,0 +1,100 @@
+//https://github.com/EnCiv/civil-pursuit/issues/51
+import RankStep from '../app/components/rank-step'
+import React from 'react'
+import expect from 'expect'
+
+import { onDoneDecorator, onDoneResult } from './common'
+import { userEvent, within } from '@storybook/testing-library'
+import Point from '../app/components/point'
+export default {
+  component: RankStep,
+  parameters: {
```

Comment:
please see point=input.stories.jsx for how to add the onDoneDecorator to these tests, and how to add a play property to a story that will test the onDone results.